### ISSUE TITLE
How to change raster postion (X-axis number)?

### ISSUE THREAD

**Issue by MrChebur** [jorgealmerio/valuetool#24]
Can I change the position of the image on the X-axis of the graph?
If not, it would be nice to make it possible.

![1](https://user-images.githubusercontent.com/20655102/159688585-7089ebd9-a937-452e-a4b1-c0168e2bf0cf.png)

**Comment by jorgealmerio** [comment]
Yes, good idea! I put an enhancement label in this issue ! Thanks!

**Comment by jfbourdon** [comment]
A small note about this:
When the Show layers combobox is set to "Visible layers", the order in which the raster layers appear in the widget is the same as the one in the Layers Panel. But when set to "Selected layers" or "All layers", the order is changed to an ascending alphabetical order.
